airmozilla/starred/static/starred/js/star_event.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready(function(){
+    $(".entry-summary").on("click", function () {
```

Comment:
I don't think this is true. If you click anywhere on the `<div class="entry-summary">` it shouldn't toggle the star.
I think what you're looking for is `$('.entry-summary').on('click', 'a.star', function() {`